releng/com.b2international.snowowl.server.update/assembly/common/scripts/snowowl_delta_import.sh

```diff
@@ -214,7 +222,7 @@ while getopts ":hu:p:f:b:a:z:U:P:" option; do
 		SOURCE_FOLDER=${OPTARG}
 		;;
 	b)
-		SNOW_OWL_BASE_URL=${OPTARG}
+		SNOW_OWL_BASE_URLS=${OPTARG}
```

Comment:
`getops` allows multi valued parameters, the only restriction is that the user needs to write the parameter letter before each possible value. E.g. **-u** http://snowowl-1.url **-u** http://snowowl-2.url
I would recommend using this solution compared to the space separated list of URLs. It will make it much more clean and terminal friendly! Here is a short example:
```
URLS=()

for URL in "${URLS[@]}"; do
	echo "${URL}"
done

while getopts "u:" opt; do
	case "$opt" in
	u)
		URLS+=("$OPTARG")
		;;
	esac
done
```